pkg/sql/schemachanger/testdata/explain_verbose/drop_table

```diff
@@ -392,14 +399,29 @@ EXPLAIN (ddl, verbose) DROP TABLE db.sc.t;
 │       │       └── • skip PUBLIC → ABSENT operations
 │       │             rule: "skip index dependents removal ops on relation drop"
 │       │
-│       └── • 11 Mutation operations
+│       └── • 12 Mutation operations
 │           │
 │           ├── • MarkDescriptorAsDropped
 │           │     DescriptorID: 107
 │           │
-│           ├── • RemoveAllTableComments
+│           ├── • RemoveTableComment
 │           │     TableID: 107
 │           │
+│           ├── • LogEvent
+│           │     Element:
+│           │       TableComment:
+│           │         comment: t has a comment
+│           │         tableId: 107
+│           │     EventBase:
+│           │       Authorization:
+│           │         UserName: root
+│           │       Statement: DROP TABLE ‹db›.‹sc›.‹t›
+│           │       StatementTag: DROP TABLE
+│           │       TargetMetadata:
+│           │         SourceElementID: 1
+│           │         SubWorkID: 1
+│           │     TargetStatus: 1
```

Comment:
I can't decide whether this is a good thing to show. It's fine for now but we might end up generating a lot of events when dropping tables.

Reply:
I'm working on this. The event logs are a bit of a nightmare to work with in the declarative schema changer. We should just roll out a new kind of event message.